### DISCUSSION TITLE
Triton to linalg Pass error

### DISCUSSION THREAD

**Discussion by LAJIidea** [microsoft/triton-shared#47] [General]
These problems occurred when I used the triton-to-linalg pass to convert nanoGPT's ttir to mlir:
1. The first one is that I found that tt.assert will cause some unsupported operations, such as arith.andi. The picture below is my error message.
![image](https://github.com/microsoft/triton-shared/assets/42372011/979a7dd7-2ca9-47b2-abb6-fe916bd05645)

2. Some unsupported operations may occur during tt.load, such as arith.cmpi or arith.divsi
![image](https://github.com/microsoft/triton-shared/assets/42372011/1efd4dad-14b3-4bae-86df-8fae75c97989)

**Comment by manbearian** [comment]
@nhat-nguyen

**Comment by nhat-nguyen** [comment]
@LAJIidea we have an issue tracking here: https://github.com/microsoft/triton-shared/issues/16
